chia/plotting/manager.py

```diff
@@ -262,18 +267,18 @@ def _refresh_task(self, sleep_interval_ms: int):
                     f"total_duration {total_result.duration:.2f} seconds"
                 )
             except Exception as e:
-                log.error(f"_refresh_callback raised: {e} with the traceback: {traceback.format_exc()}")
+                self.log.error(f"_refresh_callback raised: {e} with the traceback: {traceback.format_exc()}")
                 self.reset()
 
     def refresh_batch(self, plot_paths: List[Path], plot_directories: Set[Path]) -> PlotRefreshResult:
         start_time: float = time.time()
         result: PlotRefreshResult = PlotRefreshResult(processed=len(plot_paths))
         counter_lock = threading.Lock()
 
-        log.debug(f"refresh_batch: {len(plot_paths)} files in directories {plot_directories}")
+        self.log.debug(f"refresh_batch: {len(plot_paths)} files in directories {plot_directories}")
```

Comment:
Without adding `self.`, messages wouldn't be written in a log file. This is the reason I added `self.` around.
Could you check in your `debug.log` whether "refresh_batch: " or other log entires are appeared?

Reply:
Works fine here for me on `main` yeah. You sure the harvester was connected to a farmer and did run this code at all when you didn't see logs? And just a general thing: Even if it would not work it would be good to move something like this into a separate PR into `main` instead of sneaking it into a PR against the compression branch.